release_notes/372-release-notes.adoc

```diff
@@ -8,4 +8,265 @@ toc::[]
+** `CVE Type:NODE_CVE`
+** `CVE_Type:IMAGE_CVE`
+** `CVE_TYPE:K8S_CVE`
+* Suppress and unsuppress payloads: 
+** The field `ids` in the `/v1/cves/suppress` and `/v1/cves/unsuppress` API payloads will be renamed to `cves` in the {product-title-short} 3.73 release.
+** The `cves.ids` field of the `storage.VulnerabilityRequest` object in the response of `VulnerabilityRequestService` endpoints will be renamed to `cves.cves` in the {product-title-short} 3.73 release.
+
+[id="known-issues"]
+== Known issues
+
+{product-title-short} shows the wrong severity when two severities exist for a single vulnerability in a single distribution. This issue occurs because {product-title-short} scopes severities by namespace rather than component. There is no workaround. (ROX-12527)
+
+[id="bug-fixes-372"]
+== Bug fixes
+
+* Before this update, the steps to configure {ocp} OAuth for more than one URI were missing. The documentation has been revised to include instructions for configuring OAuth in {ocp} to use more than one URI. For more information, see xref:../operating/manage-user-access/configure-ocp-oauth.adoc#create-additional-routes-ocp-oauth_configure-ocp-oauth[Creating additional routes for the OpenShift Container Platform OAuth server]. (ROX-11296) 
+* Before this update, the autogenerated image integration (for example, a Docker registry integration) for a cluster is not deleted when the cluster is removed from Central. This issue is fixed. (ROX-9398)
+* Before this update, the `Image OS` policy criteria did not support regular expression, or `regex`. However, the documentation indicated that regular expression was supported. This issue is fixed by adding support for regular expression for the `Image OS` policy criteria. (ROX-12301)
+* Before this update, the `syslog` integration did not respect a configured TCP proxy. This is now fixed.
+* Before this update, the `scanner-db` pod failed to start when a resource quota was set for the `stackrox` namespace, because the `init-db` container in the pod did not have any resources assigned to it. The `init-db` container for `ScannerDB` now specifies resource requests and limits that match the `db` container. (ROX-12291)
+
+
+[id="image-versions-372"]
+== Image versions
+
+[%header,cols="1,1,2"]
+|===
+| Image | Description | Current version
+
+| Main
+| Includes Central, Sensor, Admission Controller, and Compliance.
+Also includes `roxctl` for use in continuous integration (CI) systems.
+| `registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8:3.72`
+
+| Scanner
+| Scans images and nodes.
+| `registry.redhat.io/advanced-cluster-security/rhacs-scanner-rhel8:3.72`
+
+| Scanner DB
+| Stores image scan results and vulnerability definitions.
+| `registry.redhat.io/advanced-cluster-security/rhacs-scanner-db-rhel8:3.72`
+
+| Collector
+| Collects runtime activity in Kubernetes or {ocp} clusters.
+| `registry.redhat.io/advanced-cluster-security/rhacs-collector-rhel8:3.72`
```

Comment:
If I expand the page big enough, these end up on a single line; Maybe use a bulleted list here?